geometry_msgs/Point[] waypoints
---
bool success